extensions/cloud/portability-cloud-local/src/main/java/org/dataportabilityproject/cloud/local/LocalCloudExtension.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Google Inc.
```

Comment:
nit: update this

Reply:
It looks like there are a bunch of these. I've added an issue 241 for this.